### PR TITLE
project: Add debian packaging metadata

### PR DESCRIPTION
**Pull request by dylanmccall**
This is copied from OBS, with the following changes:

- Removed two patches, which were applying changes that are already in this version of the source.
- Changed maintainer, homepage, etc. to something more useful.

https://phabricator.endlessm.com/T35079

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Source: force-quit-dialog-extension
+Section: gnome
+Priority: optional
+Maintainer: Dylan McCall <dylan@endlessos.org>
+Uploaders: Dylan McCall <dylan@endlessos.org>
```
Comment (wjt) [inline debian/control]: I often use:

```suggestion
Maintainer: Endless OS Maintainers <maintainers@endlessos.org>
```

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Standards-Version: 4.5.0
+Rules-Requires-Root: no
+Homepage: https://github.com/endlessm/force-quit-dialog-extension
+Vcs-Browser: https://github.com/endlessm/force-quit-dialog-extension
+Vcs-Git: https://github.com/endlessm/force-quit-dialog-extension.git
+
+Package: force-quit-dialog-extension
+Architecture: all
+Depends:
+ gnome-shell (>= 3.37.90)
```
Comment (wjt) [inline debian/control]:
```suggestion
 gnome-shell (>= 43)
```

File: debian/copyright [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Force Quit Dialog Extension
+Source: https://gitlab.gnome.org/feaneron/force-quit-dialog-extension
```
Comment (wjt) [inline debian/copyright]: Georges archived that repo: This is the upstream now.

File: debian/copyright [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Files: *
+Copyright: 2020 Georges Basile Stavracas Neto <georges@endlessm.com>
```
Comment (wjt) [inline debian/copyright]:
```suggestion
Copyright: 2020–2024 Endless OS Foundation LLC
```

**Review by wjt — CHANGES_REQUESTED** [review]

**Comment by dylanmccall** [comment]
Thanks for the review! I corrected those details :)

**Comment by wjt** [comment]
retest this please

**Comment by wjt** [comment]
```
+ dch -v 0+dev20.639a38a -D master --force-distribution 'Automatic release from git (0+dev20.639a38a)'
dch: fatal error at line 1219:
New version specified (0+dev20.639a38a) is less than
the current version number (3.38.0-1endless1)!  Use -b to force.
```

You need a `Version_XXXX` tag

**Comment by wjt** [comment]
Looks good once you it builds.

**Comment by dylanmccall** [comment]
:+1: [Ah, I see!](https://github.com/endlessm/eos-build/blob/master/ci-integration/jenkins-obs-build#L105-L123) I added a new release with the appropriate tag. Looks like it's working now :)

**Comment by dylanmccall** [comment]
Oh, I was expecting that to fastforward, but somehow not :/ That's why there's one more build failure on master, but all sorted now.

**Pull request merged**